.github/CODEOWNERS

```diff
@@ -10,5 +10,3 @@
 /src/Caching/                           @tratcher @anurse
```

Comment:
we should probably hand these off too @dotnet/extensions-migration .

Reply:
@JunTaoLuo are you able to provide details as to where the source for logging & logging abstractions now live? as the master branch no longer contains the source as far as I can tell?

Reply:
https://github.com/dotnet/runtime/blob/6ef538d46fc971f351a22cbe854717f4ada59d89/src/libraries/Microsoft.Extensions.Logging.Abstractions/src/ILogger.cs

Reply:
See https://github.com/aspnet/Announcements/issues/411 for the full list of packages and where the code lives after this migration is complete.

Reply:
@Tratcher @anurse thanks for pointing me to the announcement and new location for the source; it may be good to also update the README.md accordingly with a more information and/or to redirect others to the new location for source; as people may still think the namespace/package relates to this repo and not the runtime repo if they have missed the announcement.

Reply:
@bradygaster how far along are you with updating the readme?

Reply:
https://github.com/aspnet/Logging still points to https://github.com/dotnet/extensions which is quite confusing now

Reply:
@poizan42 did you see the updated ReadMe in extensions?

Reply:
Yes, but the description at https://github.com/aspnet/Logging should probably still be updated to point to https://github.com/dotnet/runtime/tree/master/src/libraries/Microsoft.Extensions.Logging or something